packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -353,201 +342,6 @@ describe('useMachine hook', () => {
     render(<App />);
   });
 
-  it('should capture all actions', (done) => {
```

Comment:
those removed tests were about `asEffect`+`asLayoutEffect` and those are being removed in this PR